public/index.html

```diff
@@ -1,96 +1,137 @@
+  </ul>
+  <p>Take a look at the <a href="?showSrc">page source</a> to start your own integration.</p>
+
+  <div class="dotted">
+    <h3>Here are two example points of integration.</h3>
+    <p>1. From a user filled text box</p>
+    <!-- This input box will launch the interface when the user presses enter -->
+    <input id="input-box" placeholder="Enter search query" onkeydown="inputKeyDown(event)">
     <script>
-        var getUrlParam = function(e){var t = new RegExp("[?&]" + e.replace(/[\[\]]/g, "\\$&") + "(=([^&#]*)|&|#|$)"),a = t.exec(window.location.href);return a && a[2] ? decodeURIComponent(a[2].replace(/\+/g, " ")) : ""};
-        window._sjui = { config: {
-          project: "your-project",       // Set this to your project.
-          collection: "your-collection", // Set this to your collection.
-          values: { resultsPerPage: "10" },
-          initialValues: { "q": getUrlParam("q") },
-          overlay: true,
-          pipeline: "website", // Run the website pipeline.
-          searchBoxPlaceHolder: "Type to search", // Placeholder text in search box.
-          showImages: false, // Show images in results.
-          tabFilters: { // Tabs configuration.
-            defaultTab: 'All', // Title of the default tab.
-            tabs: [{title: 'All', filter: ''}, { title: 'Blog', filter: "dir1='blog'" }, { title: 'Other', filter: "dir1!='blog'"}], // Ordered list of tabs to show.
-          },
-        }};
+      function inputKeyDown(e) {
+        if (e.keyCode === 13 /* Enter key */) {
+          window._sjui.state.setValues({ q: e.target.value }, true);
+          window._sjui.overlay.show();
+        }
+      }
     </script>
+    <p>2. From a user focusing a text box or clicking a button</p>
+    <!-- This input box will launch the interface when the user clicks it -->
+    <input onfocus="window._sjui.overlay.show()" placeholder="Focus me to launch">
+    <!-- This button will launch the interface when the user clicks it -->
+    <button onclick="myA('pub', 'overlay-show')">Launch Interface</button>
 
-    <style>
-      /**
+    <div id="search-box"></div>
+    <div id="search-response"></div>
+    <div id="search-box-2"></div>
+    <div id="search-response-2"></div>
+  </div>
+  <!-- This script sets up the config for the search interface -->
+  <script>
+    var getUrlParam = function (e) { var t = new RegExp("[?&]" + e.replace(/[\[\]]/g, "\\$&") + "(=([^&#]*)|&|#|$)"), a = t.exec(window.location.href); return a && a[2] ? decodeURIComponent(a[2].replace(/\+/g, " ")) : "" };
+    var config = (search, response) => ({
+      project: "<your-project>",       // Set this to your project.
+      collection: "<your-collection>", // Set this to your collection.
+      values: { resultsPerPage: "10" },
+      initialValues: { "q": getUrlParam("q") },
+      attachSearchBox: document.getElementById(search),
+      attachSearchResponse: document.getElementById(response),
+      overlay: search === "search-box",
```

Comment:
The #id of the element decides if this is an overlay, maybe just make this true/false, it's pretty obvious then what it means.